Cargo.toml

```diff
@@ -47,7 +47,7 @@ metal = "0.29"
 risc0-bigint2 = { version = "1.4.0", default-features = false, path = "risc0/bigint2" }
 risc0-binfmt = { version = "2.0.0", default-features = false, path = "risc0/binfmt" }
 risc0-build = { version = "3.0.0", default-features = false, path = "risc0/build" }
-risc0-build-kernel = { version = "1.4.0", default-features = false, path = "risc0/build_kernel" }
+risc0-build-kernel = { version = "2.0.0", default-features = false, path = "risc0/build_kernel" }
```

Comment:
Curious about this bump

Reply:
Sorry I forgot to mention in the PR message, this was due to my change which added `non_exhaustive` to some type in this crate.